lib/api/controllers/securityController.js

```diff
@@ -1212,6 +1213,19 @@ class SecurityController extends BaseController {
 
     return Bluebird.resolve(strategyFields);
   }
+
+  revokeTokens(request) {
+    assertHasId(request);
+
+    const userId = request.input.resource._id;
+    return this.kuzzle.repositories.user.load(userId)
+      .then(user => {
+        if (!user) {
+          errorsManager.throw('user_not_found', userId);
+        }
+        return this.kuzzle.repositories.token.deleteByUserId(userId);
```

Comment:
The documentation indicate that this route return `null` but here you return the result of `deleteByUserId`